ml_perf/reference_implementation.py

```diff
@@ -443,9 +457,23 @@ def parse_win_stats_table(stats_str, num_lines):
 
 async def train_eval(state, tf_records):
     await train(state, tf_records)
+    if FLAGS.validate:
+      await validate(state)
```

Comment:
Maybe add a TODO to run validate and eval in parallel. Although if we're moving to continuous selfplay it might not be worth bothering with.

Reply:
validate & eval both run on device 0 at the moment so i think eval has to be multi-device for this to work.